.gitignore

```diff
@@ -0,0 +1,2 @@
+venv/
+.idea/
```

Comment:
What is the .idea/ folder ? Is it specific to a file editor?

Reply:
Yes my IDE automatically creates a virtual environment and an idea folder for the project configuration. I will delete the .gitignore file at the next commit.